src/main.cpp

```diff
@@ -28,7 +28,7 @@ int main(int argc, const char **argv)
     std::map<std::string, docopt::value> args = docopt::docopt(USAGE,
       { std::next(argv), std::next(argv, argc) },
       true,// show help if requested
-      "Naval Fate 2.0");// version string
+      "Naval Fate 0.0.1");// version string
```

Comment:
Can we pull the version number in from the cmakelists file, so users don’t have to specify this twice?

Reply:
Oops, I misunderstood your comment and just reacted to the code.

I am accustomed to defining constants in my build files so I can pass them around everywhere they need to go; if this is bad practice, I probably should not give advice here. I just hate having to update version numbers in more than one place because I often get it wrong.

I am intrigued by the idea of a configured.hpp file; I’ve never seen one before. It probably makes more sense than pulling constants directly in from cmake, and letting the compiler decide what type they should be.

Reply:
I'm talking about using `configure_file` https://cmake.org/cmake/help/latest/command/configure_file.html

Reply:
That sounds like a really good idea. I wish I had known about this years ago.

Reply:
@ddalcino how does this look now?